lib/components/user/saved-trip-screen.js

```diff
@@ -229,6 +229,12 @@ const mapDispatchToProps = {
 }
 
 export default withLoggedInUserSupport(
-  withLoginRequired(connect(mapStateToProps, mapDispatchToProps)(SavedTripScreen)),
+  withAuthenticationRequired(
+    connect(mapStateToProps, mapDispatchToProps)(SavedTripScreen),
+    {
+      // Redirect back to this page if a user logs in from the URL for saving a trip.
+      returnTo: () => window.location.hash.substr(1)
+    }
```

Comment:
This object with `returnTo` is used multiple times. It should probably be encapsulated somewhere for reuse.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/233/commits/4a6c2911bce4e981f5b2162cbb4e4696cb9f4dc0